scripts/install_ebpf.psm1

```diff
@@ -335,5 +335,8 @@ function Uninstall-eBPFComponents
     Write-Log("MSI uninstallation completed successfully!") -ForegroundColor Green
 
     # Stop KM tracing.
-    wpr.exe -cancel
+    $process = Start-Process -FilePath wpr.exe -ArgumentList "-cancel" -NoNewWindow -Wait -PassThru
```

Comment:
I'd be proactive and make sure we call cancel iff we detect a current trace. Does wpr.exe return a specific exit code if we cancel a non-existent trace?

Reply:
> I'd be proactive and make sure we call cancel iff we detect a current trace. Does wpr.exe return a specific exit code if we cancel a non-existent trace?

On further thought, this seems confusing.  we need to stop the trace in order to get the .etl file, so why bother cancelling a trace that has already stopped?  The wpr handling needs to be outside of the uninstall activity, IMO.